src/safe_apps/models.py

```diff
@@ -11,7 +11,8 @@ def __str__(self):
 
 
 class SafeApp(models.Model):
-    url = models.URLField(primary_key=True)
+    app_id = models.BigAutoField(primary_key=True)
```

Comment:
FYI: Having just `id` as the name of the field conflicts with some internals regarding migration validation.

Adding a field `id` to the model results on an error stating that the field already exists (internal to Django).
Creating a custom migration to alter the field `id` to be the primary key results on an error stating that `id` does not exist.

One possible solution would be to copy the data present on the safe-apps table to a new table that sets this new `id`. But I find this solution easier to read and maintain.